src/components/try-hcp-callout/components/try-hcp-callout-compact/index.tsx

```diff
@@ -18,6 +19,7 @@ export function TryHcpCalloutCompact({
 	ctaText,
 	ctaUrl,
 }: TryHcpCalloutCompactProps) {
+	const trialCtaText = useABTestCta(ctaText)
```

Comment:
Maybe we can leave a breadcrumb here to remove this when the test is finished:
```suggestion
	// TODO: remove this when the HCP CTA Trial 2023-02 test is finished
	const trialCtaText = useABTestCta(ctaText)
```

Reply:
🧠 will do!